examples/benchmark.rs

```diff
@@ -63,6 +64,15 @@ fn client(kind: Client) {
 
 static CLIENT_DONE: AtomicBool = ATOMIC_BOOL_INIT;
 
+
+#[cfg(not(all(feature = "phy-tap_interface", 
+          all(target_os = "linux", target_env = "gnu"))))]
+fn main() {
```

Comment:
Please do not do this. Instead, add the `phy-tap_interface` feature as a dependency in `Cargo.toml`.